hepcrawl/pipelines.py

```diff
@@ -32,7 +32,8 @@ def has_publication_info(item):
         item.get('journal_fpage') or \
         item.get('journal_lpage') or \
         item.get('journal_artid') or \
-        item.get('journal_doctype')
+        item.get('journal_doctype') or \
+        item.get('pubinfo_material')
```

Comment:
The material should not be taken into account if it's only by itself, so an item that only has the material is not considered as having publication info.